GliaWidgets/Coordinator/RootCoordinator.Environment.swift

```diff
@@ -0,0 +1,23 @@
+extension RootCoordinator {
+    struct Environment {
+        var chatStorage: Glia.Environment.ChatStorage
+        var fetchFile: CoreSdkClient.FetchFile
+        var sendSelectedOptionValue: CoreSdkClient.SendSelectedOptionValue
+        var uploadFileToEngagement: CoreSdkClient.UploadFileToEngagement
+        var audioSession: Glia.Environment.AudioSession
+        var uuid: () -> UUID
```

Comment:
And this one I guess is used for when one wants to generate a UUID, to ensure that you can test it? But the name is very undescriptive.

Reply:
I added explanation https://github.com/salemove/ios-sdk-widgets/pull/346#discussion_r933312932.